cogs/send_get_roles_reminders.py

```diff
@@ -156,6 +156,13 @@ async def send_get_roles_reminders(self) -> None:
             if time_since_role_received <= datetime.timedelta(days=1):
                 continue
 
+            if member not in guild.members:
+                    logger.info(
+                        "Member: %s left the server before the introduction "
+                        "reminder could be sent.",
+                    ), member
```

Comment:
* The closing bracket must be *after* the argument `repr(member.id)` is passed into the function.
* The argument `member` is not necessary as there is only one substitution to be made: the member's ID